llvm/docs/CommandGuide/llvm-nm.rst

```diff
@@ -257,6 +257,11 @@ OPTIONS
 
  Do not filter special symbols from the output.
 
+.. option:: --synthetic
+ Include synthetic symbols in the output. Synthetic symbols are not present in
+ the original object file but are generated by the tool to represent relevant
+ runtime or linking information.
```

Comment:
Perhaps 

Include synthetic symbols in the output. Synthetic symbols are not present in the original object file. This currently includes symbols derived from PLT entries for ELF.

I find 
"are generated by the tool to represent relevant runtime or linking information." very vague and not very useful. The PLT detail seems important.